editor/script_create_dialog.cpp

```diff
@@ -35,6 +35,16 @@
 #include "os/file_access.h"
 #include "script_language.h"
 
+void ScriptCreateDialog::_notification(int p_what) {
+
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE: {
+			path_button->set_icon(get_icon("Folder", "EditorIcons"));
```

Comment:
Actually that was first thing I've tried. But instead of my icon it only rendered Error texture :[

After few failed attempts I've asked on IRC, but no one responded.
So I've investigated how it is made in other source files and it was made like this. Works but don't know why only like that :B